t80s_reduce/t80s_sort_night/sort_night_files_test.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
```

Comment:
Maybe this would be better in a test directory elsewhere. Maybe create a directory under t80s_reduce/tests/ and move it there.